src/wallets/IdentityWallet.ts

```diff
@@ -0,0 +1,179 @@
+import { ethers } from 'ethers'
+
+import { TLProvider } from '../providers/TLProvider'
+import { TLWallet } from './TLWallet'
+
+import { DeployedIdentity, UserObject } from '../typings'
+
+export class IdentityWallet implements TLWallet {
+  // TODO: make this class a TLSigner as part of https://github.com/trustlines-network/clientlib/issues/194
+
+  public provider: TLProvider
+
+  private wallet: ethers.Wallet
+  private identityAddress: string
+
+  constructor(provider: TLProvider) {
+    this.provider = provider
+  }
+
+  public get address(): string {
+    return this.identityAddress
+  }
+
+  public get pubKey(): string {
```

Comment:
This is not the right pub key. We can not implement that right now, so just throw a not implemented error.